### DISCUSSION TITLE
[2024-11-25] Libs and Engine Team Demo Day

### DISCUSSION THREAD

**Discussion by GregoryTravis** [enso-org/enso#11640] [Weekly Demo Day]
- James: New NumberParser for Table parsing #11499
- James: Linting and Widgets #11551
- James: `skip_rows` for `Excel_Workbook.read`. #11560
- James: New Table.input function #11562
- James: Fixes a few bits... #11581
- Radosław: Initial implementation of `Data.read_many` #11490
- Radosław: Support for `Table` output in `Data.read_many` #11546
- Radosław: Update `Enso_File` download to Cloud API changes #11596
- Radosław: Avoid printing expected exceptions inside HTTP tests #11607
- Greg: Create Regex benchmarks #11504
- Greg: HTTP cache size limit environment variables #11530
- Greg: Don't log stack trace in cache tests #11610
- Ad: Complete migration to Dialect/Feature flags + SQLServer support for Datetime #11527
- Ad: Enable Union feature and make tests pass #11587
- Ad: add-grab-icon #11616
- Ad: Fix Sorting in Snowflake #11636
- Martha: Table viz column header tooltip extended to add data quality stats #11520
- Martha: Add any non trivial whitespace data quality indicator #11559
- Martha: fix filter node drilldown for number columns #11572
- Martha: use equals when only one item is selected in filter dropdown #11613
- Martha: Add date support for filter drilldown #11626
- Martha: Drive data quality metrics from enso code #11638
- Jaroslav: Multi value `Complex` test and robustness refactoring #11525
- Jaroslav: Errors and completions in NetBeans 24+ #11531
- Jaroslav: Using `Ref.new allow_gc=True` to implement in-memory caches #11577
- Jaroslav: Allow (42 : Text & Integer) at the end of function #11588
- Jaroslav: Invoke Warning.getValue().to_text and use it from `ReplDebuggerInstrument` #11591
- Jaroslav: `HostValueToEnsoNode` is not a builtin #11593
- Jaroslav: Give `DebugBreakpointNode` some section to prevent NPE #11594
- Jaroslav: Pass intersection type thru the `-> Complex` check #11600
- Jaroslav: Giving TypeOfNode richer query API #11618
- Hubert: More logs to debug suspend issue #11518
- Hubert: Fix Logger's name in stdlib #11519
- Hubert: Wip/hubert/ci issues #11537
- Hubert: Revert disabling of polyglot Ydoc #11544
- Hubert: Optional `scripts` section in libraries' config #11555
- Pavel: --jvm tries to find Java executable system-wide. #11500
- Pavel: More IR mini passes #11501
- Pavel: Implement hasLanguage interop message for all enso objects #11538
- Dmitry: Build Ydoc on CI #11517
- Dmitry: Optimize runtime Docker image size #11529
- Dmitry: Add healthcheck route to Ydoc #11545
- Dmitry: Update ydoc-server build #11550
- Dmitry: Rename unqualified method calls #11556
- Dmitry: Run SBT clean before CI build #11573
- Dmitry: Deliver full intersection type of a value to the IDE #11583
- Dmitry: Fix compilation error in integration tests #11611
- Dmitry: Ydoc library #11615
- Dmitry: Fix runtime-instrument-common benchmark #11619
- Dmitry: Update message handler supervisor to watch children #11630

**Comment by radeusgd** [comment]
![image](https://github.com/user-attachments/assets/4e8aee17-5bfc-4743-a45f-208a2dfb24d9)
![image](https://github.com/user-attachments/assets/d2051788-29ef-4f31-9b6a-274ec38030f1)
